### PR TITLE
Issue/#5

### PR DESCRIPTION
**Pull request by paganpasta**
For the issue observed #5 . I made the following changes. 

- Remove custom metrics in favour of Torchmetrics' ClassificationAccuracy. Downside is that other values are not reported at the moment. This can be added based on requirement.
- Changed monitoring from `val/loss` to `val/acc` for classification
- I am not sure how much time it takes to do the running computation while training. But, it feels to me quite fast at the moment. 
- I tried with image folder builder Not the default on used in config.
- I used 2GPU setup and could only run for few iterations with reduced batch sizes.

Overall, I see metrics being computed during training and reported on the progress bar. 

Command I used: 
```bash

python src/train.py data.dataset_builder.data_dir=$DATASET_DIR seed=3407 model.network.dim=128  data.size=224 model.network.kernel_size=32 model.network.nb_layers=12 model.network.order=2 model.network.order_expand=4 model.network.ffw_expand=4  model.network.dropout=0.0 model.optimizer.optim.weight_decay=0.01 model.optimizer.optim.lr=1e-3 data.full_batch_size=1024 trainer.max_steps=300000 model.lr_scheduler.warmup_steps=10000 computer.num_workers=8 computer.precision=bf16-mixed data/additional_train_transforms=randaugment data.additional_train_transforms.randaugment_p=0.1 data.additional_train_transforms.randaugment_magnitude=6 model.train_batch_preprocess.apply_transform_prob=1.0 checkpoint_dir=$CKPT_DIR  data.dataset_builder._target_=utils.data.build_imagefolder

```

### CODE REVIEW

**Comment by davidpicard** [comment]
I tested, it works, modulo the change to "monitor: val/acc" which requires "mode: max" and not "mode :min". I'll change it myself.
